src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -17,11 +17,23 @@ public static string DownloadHousingRegressionDataset()
         => Download("https://raw.githubusercontent.com/dotnet/machinelearning/024bd4452e1d3660214c757237a19d6123f951ca/test/data/housing.txt", "housing.txt");
 
         /// <summary>
-        /// Downloads the adult dataset from the ML.NET repo
+        /// Downloads the wikipedia sentiment dataset from the ML.NET repo
         /// </summary>
         public static string DownloadSentimentDataset()
         => Download("https://github.com/dotnet/machinelearning/blob/76cb2cdf5cc8b6c88ca44b8969153836e589df04/test/data/wikipedia-detox-250-line-data.tsv", "sentiment.tsv");
 
+        /// <summary>
+        /// Downloads the adult train dataset from the ML.NET repo
+        /// </summary>
+        public static string DownloadAdultTrainDataset()
+            => Download("https://raw.githubusercontent.com/dotnet/machinelearning/master/test/data/adult.train", "adult.train");
```

Comment:
>"https://raw.githubusercontent.com/dotnet/machinelearning/master/test/data/adult.trai [](start = 24, length = 85)

It needs to include the hash, otherwise it might change overtime. 

[https://github.com/dotnet/machinelearning/blob/20c36ca769d6b1f4fc816de4ae5f46266e8edd9b/test/data/adult.test](https://github.com/dotnet/machinelearning/blob/20c36ca769d6b1f4fc816de4ae5f46266e8edd9b/test/data/adult.test)